engine/src/flutter/impeller/entity/geometry/round_superellipse_geometry.h

```diff
@@ -12,14 +12,12 @@ namespace impeller {
 
 /// Geometry class that can generate vertices for a rounded superellipse.
 ///
-/// A superellipse is an ellipse-like shape that is defined by the parameters N,
-/// alpha, and beta:
-///
-///  1 = |x / b| ^n + |y / a| ^n
-///
-/// A rounded superellipse is a square-like superellipse (a=b) with its four
-/// corners replaced by circular arcs. It replicates the `RoundedRectangle`
-/// shape in SwiftUI with corner style `.continuous`.
+/// A rounded superellipse is a shape similar to a typical rounded rectangle
+/// (`RoundRect`), but with smoother transitions between the straight sides and
+/// the rounded corners. It resembles the `RoundedRectangle` shape in SwiftUI
+/// with the `.continuous` corner style. Technically, it is created by replacing
+/// the four corners of a superellipse (also known as a Lamé curve) with
+/// circular arcs.
```

Comment:
It looks like this file is already in the repo, I'll have to check it out to see about some of the assumptions I noted above.